week5/speller/dictionary.c

```diff
@@ -0,0 +1,125 @@
+// Implements a dictionary's functionality
+#include <ctype.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+#include <strings.h>
+
+#include "dictionary.h"
+
+typedef struct node
+{
+    char word[LENGTH + 1];
+    struct node *next;
+}
+node;
+
+const unsigned int N = 26;
+
+node *table[N];
+
+unsigned int wordCount;
+unsigned int hashValue;
+
+bool check(const char *word)
+{
+    hashValue = hash(word);
+
+    node *cursor = table[hashValue];
+
+    // リンクされたリストを確認
```

Comment:
```suggestion
    // 連結リストを確認
```
「リンクされたリスト」という呼称は一般的ではないと思うので、
- 連結リスト
- Linked list
のどちらかにしたいです！

Reply:
@axtx4869 
「連結リスト」という呼称に修正しました！
再度、ご確認のほど、よろしくお願いいたします。